tests/test_ujson.py

```diff
@@ -1055,3 +1055,17 @@ def test_decode_string_utf8():
         heap = hp.heapu()
         print(heap)
 """
+
+
+@pytest.mark.parametrize(
+    "test_input",
+    [
+        ("33333333303333333333"),
+        ("18446744073709551616"),  # 64 bit
+        ("-18446744073709551616"),  # 64 bit
+        ("-80888888888888888888"),
+    ],
+)
+def test_decode_big_numeric(test_input):
+    with pytest.raises(ujson.JSONDecodeError):
+        ujson.loads(test_input)
```

Comment:
This should be merged with `test_decode_raises`, which already has some tests for big ints. I have no strong opinion on whether it should all be in one test or not, though I tend towards yes (since the test code is basically identical, just with different input).